### [CVE-2020-29651](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29651)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brighgreen)

### Description

A denial of service via regular expression in the py.path.svnwc component of py (aka python-py) through 1.9.0 could be used by attackers to cause a compute-time denial of service attack by supplying malicious input to the blame functionality.

### POC

#### Reference
No PoCs from references.

#### Github
- https://github.com/engn33r/awesome-redos-security
- https://github.com/yetingli/PoCs

